app/wizards/schools/register_ect_wizard/no_previous_ect_choices_change_programme_type_step.rb

```diff
@@ -0,0 +1,29 @@
+module Schools
+  module RegisterECTWizard
+    class NoPreviousECTChoicesChangeProgrammeTypeStep < ProgrammeTypeStep
+      def next_step
+        return :check_answers if ect.school_led?
+        return :no_previous_ect_choices_change_lead_provider if school.programme_choices? || was_school_led || ect.lead_provider_id.nil?
+
+        :check_answers
+      end
+
+      def previous_step
+        return :no_previous_ect_choices_change_lead_provider if ect.provider_led? && (was_school_led || ect.lead_provider_id.nil?)
+        return :no_previous_ect_choices_change_independent_school_appropriate_body if school.programme_choices? && school.independent?
+        return :no_previous_ect_choices_change_state_school_appropriate_body if school.programme_choices? && school.state_funded?
+
+        :check_answers
+      end
```

Comment:
```suggestion
      def next_step
        return :no_previous_ect_choices_change_lead_provider if ect.provider_led?

        :check_answers
      end

      def previous_step
        return :no_previous_ect_choices_change_independent_school_appropriate_body if school.independent?
        
        :no_previous_ect_choices_change_state_school_appropriate_body
      end
```

Reply:
I think we agreed with Emily to put the guard to force the user to choose a lead provider at the top of the journey instead of in the step where they choose the programme type?

Reply:
Right after, Emily replied that it makes sense to keep the current behaviour, but she also mentioned it doesn't matter too much which behaviour we go with.

We should get more feedback after the product review and can adjust it accordingly.